tasmota/xdrv_04_light.ino

```diff
@@ -1844,6 +1844,16 @@ uint16_t fadeGammaReverse(uint32_t channel, uint16_t vg) {
   }
 }
 
+uint8_t LightGetCurFadeBri(void) {
```

Comment:
This assumes the brightness is calculating by taking the maximum brightness of each channel.